src/web/handlers.rs

```diff
@@ -31,148 +32,177 @@ pub const ONE_KB: f64 = 1024.0;
 pub async fn get_collections(
     meta: MetaRequest,
     db_pool: DbTransactionPool,
+    request: HttpRequest,
 ) -> Result<HttpResponse, Error> {
     db_pool
-        .transaction_http(|db| async move {
-            meta.metrics.incr("request.get_collections");
-            let result = db.get_collection_timestamps(meta.user_id).await?;
-
-            Ok(HttpResponse::build(StatusCode::OK)
-                .header(X_WEAVE_RECORDS, result.len().to_string())
-                .json(result))
-        })
+        .transaction_http(
```

Comment:
can we make `request` the first argument to `transaction_http` instead? It kind of gets lost to my eyes since we pass the large closure inline everywhere in this file. I think it'll significantly reduce the diff from master too.

Reply:
(something, something, add arguments to the end of the list to avoid unintentional value setting, something, something...)